src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -11415,4 +11415,92 @@ public  void    M   (   )
             }
             """,
             parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp14));
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/13163")]
+    public Task BlockFollowedByParenthesizedExpression()
+        => AssertFormatAsync("""
+            class C
+            {
+                void M()
+                {
+                    { }
+                    (0).ToString();
+                }
+            }
+            """, """
+            class C
+            {
+                void M()
+                {
+                    { }
+                     (0).ToString();
```

Comment:
I kept asking myself why we wanted a 5 space indentation after formatting only to look it up and this is the code prior to formatting. 🤦